openshift/datasync-http.yml

```diff
@@ -2,22 +2,21 @@
 kind: Template
 apiVersion: v1
 metadata:
-  name: datasync-http
+  name: datasync-server-app
   annotations:
-    openshift.io/display-name: Mobile Developer Services Data Sync Gateway
-    description: Mobile Developer Services DataSync template
+    openshift.io/display-name: Data Sync App
+    description: Template allows to deploy DataSync Server application from source code
```

Comment:
"This template contains a DataSync Server application."

Another line explaining it's purpose would be beneficial IMO.

Same for parameter "openshift.io/long-description:"